services/alertmanager/alertmanager.go

```diff
@@ -37,21 +37,21 @@ import (
 const (
 	alertmanagerDataDir = "/srv/alertmanager/data"
 	prometheusDir       = "/srv/prometheus"
-	path                = "/srv/alertmanager/alertmanager.base.yml"
+	dirPerm             = os.FileMode(0775)
+
+	path = "/srv/alertmanager/alertmanager.base.yml"
```

Comment:
That short name was originally a local constant. I think we should find a better (slightly) longer name for package-level constant.